Header Header
int32 id
float64 mid_x
float64 mid_y
float64 dist
geometry_msgs/Point[] points